akka-http-core/src/main/scala/akka/http/ClientConnectionSettings.scala

```diff
@@ -36,6 +40,7 @@ object ClientConnectionSettings extends SettingsCompanion[ClientConnectionSettin
       c getFiniteDuration "connecting-timeout",
       c getPotentiallyInfiniteDuration "idle-timeout",
       c getIntBytes "request-header-size-hint",
+      Randoms.SecureRandomInstances, // can currently only be overridden from code
```

Comment:
can stay as-is currently (not sure what one might want to substitute here other than the secure random that this yields), maybe make this a `// TODO` comment though, so it's easier to find.